### DISCUSSION TITLE
organization as default folder

### DISCUSSION THREAD

**Discussion by sadden3194** [dani-garcia/vaultwarden#3550] [General]
Hi all,

I am using vaultwarden for my wife and me (each of us has a own vaultwarden account).
we would like to see the passwords from each other.

For that I created a organization which a collection Person A and Person B.
If I create a new password it would be automatically save into my own safe.
I have to set the options manually for "save in organization" and "collection".

Is there an option, to set the option "save in organization" as default?

I am using vaultwarden (Version 2023.3.0) as a docker container on my Synology NAS

**Comment by BlackDex** [comment]
You could activate https://bitwarden.com/help/policies/#remove-individual-vault, but that will not enforce when you are admin or owner of the org. 

There are no other options as far as i know. This all btw should be handled client side.

**Reply by sadden3194** [reply]
hmmmm this could be an option but not very good :/
and the collection have to select manually, too.

other idea:
my wife and I could only use one account. this could work, isnt it?

**Reply by Guarmanda** [reply]
we need this option too, and we're both owner of the organisation :')
